installation from source issues

I attempted to install vertigo via source exactly as detailed, it seemed as if the packages where needed to be installed first. So I then attempted `go get github.com/9uuso/vertigo` then later cloned the source and ran `godep go build` and `PORT="80" MARTINI_ENV="production" ./vertigo` which everything then seemed to run as planned. Maybe `go get github.com/9uuso/vertigo` should be added alongside of `go get github.com/tools/godep` in the readme?

@9uuso I seem to get this error everytime, 

```
main.go:9:2: cannot find package "github.com/9uuso/vertigo/databases/gorm" in any of:
	/home/netmml/.gvm/gos/go1.4/src/github.com/9uuso/vertigo/databases/gorm (from $GOROOT)
	/home/netmml/Desktop/vertigo/Godeps/_workspace/src/github.com/9uuso/vertigo/databases/gorm (from $GOPATH)
	/home/netmml/Desktop/go/src/github.com/9uuso/vertigo/databases/gorm
main.go:10:2: cannot find package "github.com/9uuso/vertigo/misc" in any of:
	/home/netmml/.gvm/gos/go1.4/src/github.com/9uuso/vertigo/misc (from $GOROOT)
	/home/netmml/Desktop/vertigo/Godeps/_workspace/src/github.com/9uuso/vertigo/misc (from $GOPATH)
	/home/netmml/Desktop/go/src/github.com/9uuso/vertigo/misc
main.go:11:2: cannot find package "github.com/9uuso/vertigo/routes" in any of:
	/home/netmml/.gvm/gos/go1.4/src/github.com/9uuso/vertigo/routes (from $GOROOT)
	/home/netmml/Desktop/vertigo/Godeps/_workspace/src/github.com/9uuso/vertigo/routes (from $GOPATH)
	/home/netmml/Desktop/go/src/github.com/9uuso/vertigo/routes
main.go:12:2: cannot find package "github.com/9uuso/vertigo/settings" in any of:
	/home/netmml/.gvm/gos/go1.4/src/github.com/9uuso/vertigo/settings (from $GOROOT)
	/home/netmml/Desktop/vertigo/Godeps/_workspace/src/github.com/9uuso/vertigo/settings (from $GOPATH)
	/home/netmml/Desktop/go/src/github.com/9uuso/vertigo/settings
godep: go exit status 1
```

also my go version is 1.4.2 and os is ubuntu but does also does same on windows 8

--- a/Dockerfile
+++ b/Dockerfile
@@ -4,7 +4,7 @@ FROM golang
 
 ADD . /go/src/github.com/9uuso/vertigo
 
-RUN go get github.com/tools/godep && cd /go/src/github.com/9uuso/vertigo && godep go build
+RUN go get github.com/tools/godep && go get -u github.com/9uuso/vertigo && cd /go/src/github.com/9uuso/vertigo && godep go build
 
 WORKDIR /go/src/github.com/9uuso/vertigo
 

--- a/README.md
+++ b/README.md
@@ -46,7 +46,7 @@ For advanced usage, see [Advanced Heroku deployment](https://github.com/9uuso/ve
 ###Source
 
 1. Install Go (I recommend using [gvm](https://github.com/moovweb/gvm))
-2. `go get github.com/tools/godep`
+2. `go get github.com/tools/godep && go get -u github.com/9uuso/vertigo`
 3. `git clone https://github.com/9uuso/vertigo`
 4. `cd vertigo && godep go build`
 5. `PORT="80" MARTINI_ENV="production" ./vertigo`

